clients/python/test/e2e/ci/generate_html_table.py

```diff
@@ -4,30 +4,29 @@
 import pytest
 import typer
 from _utils import E2eExitCodes
+from postprocess_e2e import exit_code_valid
 
 
 def exitcode_to_text(exitcode: int) -> str:
     """Turn exitcodes to string"""
-    if exitcode == E2eExitCodes.INCOMPATIBLE_CLIENT_SERVER:
-        return "incompatible"
-    elif exitcode == pytest.ExitCode.OK:
-        return "pass"
-    elif exitcode == pytest.ExitCode.TESTS_FAILED:
-        return "fail"
+    if exitcode in set(E2eExitCodes):
+        return E2eExitCodes(exitcode).name
+    elif exitcode in set(pytest.ExitCode):
+        return pytest.ExitCode(exitcode).name
     else:
         raise typer.Exit(code=E2eExitCodes.CI_SCRIPT_FAILURE)
 
 
 def make_pretty(entry: str):
     color: str
-    if entry == "incompatible":
+    if entry == E2eExitCodes.INCOMPATIBLE_CLIENT_SERVER.name:
```

Comment:
I always disliked how sometimes enums mismatch strings like in here. 
I did some tests of alternatives in https://github.com/pcrespov/sandbox-python/blob/python-stuff/test_enumerations.py and concluded that in this case, the best is to have enums than also inherit from `str`
```python
from enum import Enum

class MyEnumStr(Enum, str):
    FOO = "FOO"
```